### ISSUE TITLE
Feature request: Copy settings to teams on different projects

### ISSUE THREAD

**Issue by kaizer777** [ALM-Rangers/azure-boards-kanban-tools#21]
Currently the tool is limited to copying settings between teams on a single project. It would be very useful if it could copy to team boards on other projects so that everything can be kept consistent across multiple projects.

**Comment by jessehouwing** [comment]
Agreed. There are some issues with the fact that we can't guarantee tags, iterations, areas, fields and other things even exist on the other project making the mapping a LOT more complex. Hence why we didn't do this for V1.

**Comment by jessehouwing** [comment]
#8 is the way we propose to handle these scenarios to prevent having to do it 'live'.

**Comment by amontano-ona** [comment]
Import/export would be great. We could tweak the JSON as necessary to satisfy the destination board, provided the import gave enough detail in the event of a failure.

**Comment by jessehouwing** [comment]
That would indeed be step one.

**Comment by joebieker916** [comment]
Hello,

Is this currently being worked on? If so, please provide ETA if you know.  Really need this feature ASAP.  please let me know. Thank you.

**Comment by jessehouwing** [comment]
No this extension is currently in maintenance mode and this feature requires massive investments unfortunately.

I'd love for it to be otherwise, but we can only spend our time once and this is currently not where anyone is spending it.

I'm open to accepting a pull request, but you'll likely encounter the following issues:

 * the selected process has to be 100% the same between projects.
 * security groups, users, tags, iteration paths and area paths will need to be mapped while applying these changes.
 * the current mechanism relies on directly updating the board without any intermediate format. You'll likely need to capture the settings in an intermediary format (json?) and then apply it to the target project from there.

While copying between projects was in the original scope for. This extension, it was scrapped due to there reasons and left when there was enough interest. Very few requests have come in thus far.

**Comment by nnikor** [comment]
Is still on the level of a request or has something been done to develop this? I'm one of the user who would really appreciate the feature to copy board settings between different projects due to a wide range of different stylings used and many different new projects starting continuously.
